node-operators/block-producers/operational-tasks/proxy-accounts.md

```diff
@@ -0,0 +1,161 @@
+
+A pop-up will appear where you can view an overview of all of your proxy accounts.
+
+![Review your proxy accounts.](/images/node-operators/block-producers/operational-tasks/proxy/proxy-block-7.webp)
+
+## Executing a Proxy Transaction {: #executing-a-proxy-transaction }
+
+Now that you have created a proxy account and verified that it was successfully set up, you can execute a transaction using the staking proxy account on behalf of your block producer account, also known as the primary account or the account that is being proxied. The following example will demonstrate initiating a delegation. The proxy configuration shown is a realistic example of how you might have your own proxy configured for your block producer primary account.
+
+To execute a transaction, you can navigate back to the [**Extrinsics** page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank} and take the following steps:
+
+1. Select the proxy account to submit the transaction from the **using the select account** dropdown
+2. From the **submit the following extrinsic** menu, select **proxy**
+3. Choose the **proxy** extrinsic
+4. Choose **Id** from the **AccountIdLookupOf** dropdown
+5. Select the primary account from the **real** dropdown
+6. Select the **pooledStaking** call
+7. Choose the **requestDelegate** extrinsic
+8. Enter the **candidate** address, which is the block producer account
+9. Select **AutoCompounding** for the **pool** field
+10. Enter the amount to stake. This amount must meet the minimum, which is {{ networks.dancebox.block_producers.min_self_del.dance }} DANCE tokens. You'll need to submit the value in [Planck](https://wiki.polkadot.network/docs/learn-DOT#the-planck-unit){target=\_blank}, so for {{ networks.dancebox.block_producers.min_self_del.dance }}, you'll need to enter `{{ networks.dancebox.block_producers.min_self_del.planck }}`
+11. Click **Submit Transaction**
+
+![Execute a proxy transaction from the Extrinsics page of Polkadot.js Apps.](/images/node-operators/block-producers/operational-tasks/proxy/proxy-block-8.webp)
+
+A pop-up will appear for you to authorize and sign the transaction. Enter your password for the proxy account and click **Sign and Submit**. To confirm the delegation request, you'll need to execute the pending request after two sessions have passed. Please refer to the instructions for [executing pending requests](/node-operators/block-producers/onboarding/account-setup/#execute-pending-request){target=\_blank} for a step-by-step guide. After executing the pending request, you can [verify that your block producer is in the list of eligible candidates](/node-operators/block-producers/onboarding/account-setup/#verify){target=\_blank}.
+
+That's it! You've successfully executed a transaction using a proxy account on behalf of your primary account.
+
+## Removing a Proxy Account {: #removing-a-proxy-account }
```

Comment:
ahh, ok. I'd say this section can just be in the generic Proxy Account page, and then in the Offboarding / Decommission page, we can add a note saying if you set up your block producer account with a proxy, check out the generic guide to learn how to remove it.